### ISSUE TITLE
RM-ANOVA insufficient error message for failure to calculate sphericity test

### ISSUE THREAD

**Issue by MichlF** [jasp-stats/jasp-issues#1291]
* JASP version: 14
* OS name and version: Win10
* Analysis: 
* Bug description: I have an ANOVA with two RM factors with six levels each. When checking the box for Sphericity tests, it says "Cannot perform sphericity tests because there are only two levels of the RM factor". If I delete at least one of the levels for either factor, it works. If I delete one level of each factor it also works.
* Expected behaviour: Should calculate the test.

#### Steps to reproduce:
Conduct sphericity tests on provided .csv data set using two factors (A and B) and six levels each. Ignore variable "subject_nr". Use the following first six columns as first combination (Factor A, 1. Level, Factor B, 1. through 6. Level), the next six columns as the second combination (i.e. Factor A, 2. Level, Factor B, 1. through 6. Level), and so on.
See also provided .JASP file.

[JASP_category_across_cue.zip](https://github.com/jasp-stats/jasp-issues/files/6447619/JASP_category_across_cue.zip)

**Comment by JohnnyDoorn** [comment]
Hi @MichlF,

I tried to run the analysis in R (using the afex/car package), and get the following error:
```
Singular error SSP matrix:
non-sphericity test and corrections not available
```
I think this error has to do with having too many levels, compared to the number of observations. For the RM ANOVA, this is ok, but not for the multivariate follow-up tests. Another, more technical, way of saying this is that your design is not full rank. I am no expert in this, but you can read more about this in[ this discussion](https://stats.stackexchange.com/questions/35071/what-is-rank-deficiency-and-how-to-deal-with-it).

The error message should be more informative, however. 

Kind regards,
Johnny

**Comment by tomtomme** [comment]
Tested with 0.18.3 and just works now without error message. Hence closing.
Feel free to reopen, if not fixed on your side with up to date jasp 0.18.3
